test/unit/report_importer_test.rb

```diff
@@ -19,54 +19,4 @@ class ReportImporterTest < ActiveSupport::TestCase
     r = Report.import read_json_fixture('report-no-logs.json')
```

Comment:
As `ReportImporter` is also a parent class, I am not sure if we should move this test to config_report_importer_test or use `ConfigReport.import`